build/webpack/webpack.extension.web.config.js

```diff
@@ -87,7 +87,7 @@ const config = {
             }
         ]
     },
-    externals: ['vscode', 'commonjs', 'electron'], // Don't bundle these
+    externals: ['vscode', 'commonjs', 'electron', 'moment'], // Don't bundle these
```

Comment:
Is it possible to do something to prevent we don't end up in a situtation where tomorrow we pull in a 3rd party package or an existing 3rd party package ends up pulling in moment, then things will not work, or it will fail only in certian cases based on where and how the 3rd party package ends up using moment.

E.g. can we write up a linter to look at package-lock.json and ensure that nothing ever pulls in moment except for jupyter. Else I feel this is a risky change (as mentioned if some other package pulls it in, then things could break sometimes)